docs/developer-guide/base-maps/using-with-maplibre.md

```diff
@@ -0,0 +1,248 @@
+<link href="https://unpkg.com/maplibre-gl@^3.0.0/dist/maplibre-gl.css" rel="stylesheet" />
+<script type="text/javascript">
+  const {DeckGL, ScatterplotLayer} = deck;
+
+  maplibregl.accessToken = '<mapbox_access_token>';
+
+  new DeckGL({
+    mapStyle: 'https://basemaps.cartocdn.com/gl/positron-gl-style/style.json',
+    initialViewState: {
+      longitude: 0.45,
+      latitude: 51.47,
+      zoom: 11
+    },
+    controller: true,
+    layers: [
+      new ScatterplotLayer({
+        id: 'deckgl-circle',
+        data: [
+          {position: [0.45, 51.47]}
+        ],
+        getPosition: d => d.position,
+        getFillColor: [255, 0, 0, 100],
+        getRadius: 1000
+      })
+    ]
+  });
+</script>
+```
+
+  </TabItem>
+
+  <TabItem value="react" label="React">
+
+```tsx
+import React from 'react';
+import {Map} from 'react-map-gl/maplibre';
+import DeckGL from '@deck.gl/react';
+import {ScatterplotLayer} from '@deck.gl/layers';
+import 'maplibre-gl/dist/maplibre-gl.css';
+
+function App() {
+  const layers: [
+    new ScatterplotLayer({
+      id: 'deckgl-circle',
+      data: [
+        {position: [0.45, 51.47]}
+      ],
+      getPosition: d => d.position,
+      getFillColor: [255, 0, 0, 100],
+      getRadius: 1000,
+    })
+  ];
+
+  return (
+    <DeckGL
+      initialViewState={{
+        longitude: 0.45,
+        latitude: 51.47,
+        zoom: 11
+      }}
+      controller
+      layers={layers}
+    >
+      <Map mapStyle="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json" />
+    </DeckGL>
+  );
+}
+```
+
+  </TabItem>
+</Tabs>
+
+
+## Additional Information
+
+### react-map-gl
+
+[react-map-gl](https://github.com/visgl/react-map-gl) is a React wrapper around maplibre-gl maintained by the vis.gl community. If you'd like to use deck.gl together with maplibre-gl and React, this library is the recommended companion.
+
+All the [examples on this website](https://github.com/visgl/deck.gl/tree/master/examples/website) are implemented using the React integration.
+
+When you choose the interleaved or overlaid option, the react-map-gl [Map](https://visgl.github.io/react-map-gl/docs/api-reference/map) React component acts as the root component, and [MapboxOverlay](../../api-reference/mapbox/mapbox-overlay#using-with-react-map-gl) is used with react-map-gl's `useControl` hook. 
+
+When you choose the reverse-controlled option, the `DeckGL` React component cats as the root component, and the react-map-gl [Map](https://visgl.github.io/react-map-gl/docs/api-reference/map) is a child. In this case, `Map` will automatically interpret the deck.gl view state (i.e. latitude, longitude, zoom etc), so that deck.gl layers will render as a synchronized geospatial overlay over the underlying map.
+
+
+### Choosing a map tile service
+
+There are paid map tile servers such as [MapTiler](https://www.maptiler.com/) and [AWS Location Service](https://docs.aws.amazon.com/location/latest/developerguide/map-concepts.html).
```

Comment:
There are more tiles providers/servers. I'm not sure which are entitled to be here, but stadia maps comes to mind.